### PR TITLE
Add consensus from first meeting re communication. Closes #22

### LINKED ISSUE #22: We communicate effectively
We talk with DevOps early and often.  Especially regarding Machines and Security issues

We communicate regarding deployment of code with our team, users and stakeholders

We proactively report bugs in others' code
We enable and encourage others to report bugs in our code
We do so respectfully and with justification
We prefer solutions, especially as PRs, rather than complaints

We report bugs upstream from DLSS 
If possible, we provide solutions, especially as PRs, when doing so or once the issue has been acknowledged
We use github issues in our repos to link to and track upstream bugs that are out of our control

We talk to the Design team as early as possible for UI components and accessibility

### PR DESCRIPTION
**Pull request by azaroth42**

### CODE REVIEW

**Comment by darrenleeweber** [comment]
:+1:

**Comment by mejackreed** [comment]
:+1:

File: best-practices/communication.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Department Communication
+
+We communicate early and often with other teams that will be affected by our work, or that we need input from on it.
+
+In particular,
+
+ - We talk with DevOps, particularly concerning deployment.
+ - We talk with Design, particularly concerning UI components and accessibility.
+ - We talk with Managers, particularly concerning scheduling.
```
Comment (ndushay) [inline best-practices/communication.md]: and scope?

**Comment by ndushay** [comment]
:+1:   my line note is a suggestion, not required for my vote.

**Comment by cbeer** [comment]
:+1:

**Pull request merged**